crowbar_framework/app/assets/stylesheets/bootstrap/_type.scss

```diff
@@ -232,64 +215,62 @@ abbr[data-original-title] {
   cursor: help;
   border-bottom: 1px dotted $abbr-border-color;
 }
-.initialism {
+abbr.initialism {
   font-size: 90%;
-  @extend .text-uppercase;
+  text-transform: uppercase;
 }
 
 // Blockquotes
 blockquote {
   padding: ($line-height-computed / 2) $line-height-computed;
   margin: 0 0 $line-height-computed;
-  font-size: $blockquote-font-size;
   border-left: 5px solid $blockquote-border-color;
-
-  p,
-  ul,
-  ol {
-    &:last-child {
-      margin-bottom: 0;
-    }
+  p {
+    font-size: ($font-size-base * 1.25);
+    font-weight: 300;
+    line-height: 1.25;
   }
-
-  // Note: Deprecated small and .small as of v3.1.0
-  // Context: https://github.com/twbs/bootstrap/issues/11660
-  footer,
-  small,
-  .small {
+  p:last-child {
+    margin-bottom: 0;
+  }
+  small {
     display: block;
-    font-size: 80%; // back to default font-size
     line-height: $line-height-base;
     color: $blockquote-small-color;
-
     &:before {
-      content: '\2014 \00A0'; // em dash, nbsp
+      content: '\2014 \00A0'; // EM DASH, NBSP
     }
   }
-}
 
-// Opposite alignment of blockquote
-//
-// Heads up: `blockquote.pull-right` has been deprecated as of v3.1.0.
-.blockquote-reverse,
-blockquote.pull-right {
-  padding-right: 15px;
-  padding-left: 0;
-  border-right: 5px solid $blockquote-border-color;
-  border-left: 0;
-  text-align: right;
-
-  // Account for citation
-  footer,
-  small,
-  .small {
-    &:before { content: ''; }
-    &:after {
-      content: '\00A0 \2014'; // nbsp, em dash
+  // Float right with text-align: right
+  &.pull-right {
+    padding-right: 15px;
```

Comment:
Properties should be ordered border-left, border-right, padding-left, padding-right